api/routes.go

```diff
@@ -77,8 +77,10 @@ func (api *API) buildHTTPRoutes() {
 	api.staticRouter.POST("/user/recover/request", api.WithDBSession(api.noAuth(api.userRecoverRequestPOST)))
 	api.staticRouter.POST("/user/recover", api.WithDBSession(api.noAuth(api.userRecoverPOST)))
 
-	api.staticRouter.POST("/stripe/webhook", api.WithDBSession(api.noAuth(api.stripeWebhookPOST)))
+	api.staticRouter.POST("/stripe/billing", api.WithDBSession(api.withAuth(api.stripeBillingPOST, false)))
```

Comment:
I was afraid it would make the call too verbose but I see value in a named constant. The bare `false` doesn't tell us much.
The constant name will need to be more or less descriptive, such as `acceptAPIKey`/`noAPIKey`. 

I'll keep this in mind but it's out of scope for this PR.